figo.net/FigoSession.cs

```diff
@@ -29,49 +29,68 @@ public class FigoSession {
         /// The timeout for a API request
         /// </summary>
         private int _timeout = 5000;
-        public int Timeout {
+        public int Timeout
+        {
             get { return _timeout; }
             set { _timeout = value; }
         }
 
         #region Request Handling
-        protected async Task<T> DoRequest<T>(string endpoint, string method = "GET", object body = null) {
+        protected async Task<T> DoRequest<T>(string endpoint, string method = "GET", object body = null)
+        {
             string request_body = null;
-            if(body != null)
+            if (body != null)
                 request_body = JsonConvert.SerializeObject(body, Formatting.None, new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
 
             string response_body = await DoRequest(endpoint, method, request_body);
-            if(response_body == null)
+            if (response_body == null)
                 return default(T);
-            else
-                return JsonConvert.DeserializeObject<T>(response_body);
+
+
+            //WORKAROUND für PurchasePrice Bug bei Securites
```

Comment:
what is this bug? sounds like this should not be here, but the bug should be fixed...